src/ua/dymohlo/arraylist/ArrayList.java

```diff
@@ -0,0 +1,77 @@
+package ua.dymohlo.arraylist;
+/*
+Implement ArrayList
+*/
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class ArrayList<T> {
+    private Object[] array;
+    private int size;
+
+    public ArrayList() {
```

Comment:
We should initialize size via constructor
Please add ArrayList(int size) constructor